docs/user-guide/ze-install.md

```diff
@@ -113,6 +112,24 @@ When environment conditions do not support the Zowe CLI built-in Credential Mana
 
     If the `autoStore` setting is set to *false*, you are prompted for the missing credentials in Visual Studio Code. These are stored and used for the duration of the session.
 
+### Setting confirmation requirements for submitting jobs
+
+Submitting the wrong job can risk potential problems on your server. This can happen when the user enters the wrong job or inadvertently selects the **Submit Jobs** option.
+
+To help prevent this, enable the option to require confirmation before submitting a job. Once enabled, a dialog window asking for user confirmation displays when **Submit Jobs** is selected.
+
+![Confirm Submit Job](../images/ze/ZE-SubmitJobConfirm.gif)
+
+To configure confirmation settings for submitting a job, follow these steps:
+
+1. On the VS Code menu bar, click **File**, **Preferences**, and click **Settings** to display the Settings editor.
+
+2. Select the **User** or **Workspace** tab, depending on the settings you want to update.
+3. In the Settings navigation menu, open the **Extensions** menu and click **Zowe Explorer**.
+4. Select **Jobs: Confirm Submission** to enable or disable the Confirm Submission feature.
```

Comment:
Sorry, I should have clarified that I removed the checkbox when I added those drop-down options. We might want to update this step in accordance w/ the new options:

![image](https://user-images.githubusercontent.com/86500639/211400642-19abab19-9236-4c00-b878-e4255f4400fb.png)

Maybe we can adjust step 4 to say something like "Select an option for the **Jobs: Confirm Submission** extension setting to change the desired output for the confirmation dialog" ?